src/font.js

```diff
@@ -143,6 +142,24 @@ Font.prototype.charToGlyph = function(c) {
     return glyph;
 };
 
+/**
+ * Update features
+ * @param {any} options features options
+ */
+Font.prototype.updateFeatures = function (options) {
+    // TODO: update all features options not only 'latn'.
```

Comment:
Do you plan to finish this TODO before merging this?

Reply:
Maybe later. But it'll not affect the current functionality at all. I need find a common model for updating features on UI controls changes / passed options to stringToGlyphs for all scripts.